### DISCUSSION TITLE
Intercepting https request

### DISCUSSION THREAD

**Discussion by ghost** [mitmproxy/mitmproxy#6626] [Q&A]
I am writing a python code that will intercept all of the https requests that are being sent from my computer anyone knows how can I do this?

**Comment by emanuele-em** [comment]
What you are looking for are _Addons_

Here you can read how to create a simple Addon:
[https://docs.mitmproxy.org/stable/addons-overview/](https://docs.mitmproxy.org/stable/addons-overview/)
while here there are a lot of examples:
[https://docs.mitmproxy.org/stable/addons-examples/](https://docs.mitmproxy.org/stable/addons-examples/)
